spec/features/clearance/user_signs_up_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature "User signs up" do
+  scenario "by navigating to the page" do
+    visit sign_in_path
+
+    click_link t("sessions.form.sign_up")
+
+    expect(current_path).to eq sign_up_path
```

Comment:
Feature specs test the page from the user's perspective. Users don't care about the URL, they just want to see the expected values on the page.

I general, I try to always assert on the UI rather than the URL or the database in feature specs.